build-image/openhabian.conf

```diff
@@ -12,8 +12,10 @@ system_default_locale="en_US.UTF-8"
 #timezone=Europe/Berlin
 
 # Wi-Fi settings. An ethernet connection is recommended
-# Requirement: RPi3, RPi0W, PineA64 or a supported external Wi-Fi dongle
+# Requirement: RPi4, RPi3, RPi0W or a supported external Wi-Fi dongle
 # Fill in your SSID and password below, leave empty to use ethernet
+# ATTENTION: you need to escape these special characters: $, `, ", \, (newline)
+# 'Escaping' means to put an additional \ in front of that character
```

Comment:
I think this needs to stay here since we need to write down a properly escaped bash string even though wpa_passphrase will handle the actual writing to wpa_supplicant.conf....
It will anyway not handle special characters in SSID, so entering " in SSID will break the config file, \" should be fine.

Reply:
ok so we'll keep this new comment